api/endpoint-tests/activities/get.js

```diff
@@ -27,14 +27,14 @@ tap.test('activities endpoint | GET /activities', async getUsersTest => {
     validTest.same(
       body,
       [
-        { id: 1, name: 'view-roles' },
-        { id: 2, name: 'create-roles' },
-        { id: 3, name: 'edit-roles' },
-        { id: 4, name: 'delete-roles' },
-        { id: 5, name: 'view-users' },
-        { id: 6, name: 'add-users' },
-        { id: 7, name: 'view-activities' },
-        { id: 8, name: 'edit-users' }
+        { id: 1001, name: 'view-roles' },
+        { id: 1002, name: 'create-roles' },
+        { id: 1003, name: 'edit-roles' },
+        { id: 1004, name: 'delete-roles' },
+        { id: 1005, name: 'view-users' },
+        { id: 1006, name: 'add-users' },
+        { id: 1007, name: 'view-activities' },
+        { id: 1008, name: 'edit-users' }
```

Comment:
Set these IDs high so that they won't conflict with the database's autoincrement values in the tests that involve inserting new rows.  As long as we don't insert 1000 rows during our tests, these should be fine.  We also reset the database at the start of each set of tests, so that should give us lots and lots of wiggle room.

Reply:
nice